packages/input-option/src/InputOption/InputOption.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { InputOption } from '..';
+
+describe('packages/input-option', () => {
+  test('isPolymorphic', () => {
+    expect(InputOption).toBePolymorphic();
+  });
+
+  test('renders', () => {
```

Comment:
the only other prop we could test here is disabled -- should we add a quick test?

Reply:
What would we be testing for though? There are no baked-in functional impacts of the `disabled` prop, only visual

Reply:
we don't need to set aria-disabled on the container?

Reply:
I don't think so. The input itself has the `readOnly` attribute but should still be focusable.

Also, trying to add this to the container gives: 
> The attribute aria-disabled is not supported by the role search